examples/helm/README.md

```diff
@@ -0,0 +1,120 @@
+# jitsi-meet
+
+[jitsi-meet](https://jitsi.org/jitsi-meet/) Secure, Simple and Scalable Video Conferences that you use as a standalone app or embed in your web application.
+
+## TL;DR;
+
+```console
+$ git clone https://github.com/jitsi/docker-jitsi-meet
+$ helm install ./docker-jitsi-meet/examples/helm
+```
+
+## Introduction
+
+This chart bootstraps a jitsi-meet deployment, like the official [one](https://meet.jit.si).
+
+## Different topology
+
+To be able to do video conferencing with other people, the jvb component should be reachable by all participants (eg: a public IP).
+Thus the default behaviour of advertised the internal IP of jvb, is not really suitable in many cases.
+Kubernetes offers multiple possibilities to work around the problem. Not all options are available depending on the Kubernetes cluster setup.
+The chart tries to make all options available without enforcing one.
+
+### Option 1: Loadbalancer
+
+This requires a cloud setup that enables a Loadbalancer attachement.
+This could be enabled via values:
+
+```yaml
+jvb:
+  service:
+    type: Loadbalancer
```

Comment:
```suggestion
    type: LoadBalancer
```

There is a typo here for the service type

Reply:
Shit, I already tried to fix this typo, replaced by another typo.